proto/viam/app/v1/app.proto

```diff
@@ -745,6 +745,25 @@ enum FragmentVisibility {
   FRAGMENT_VISIBILITY_PUBLIC_UNLISTED = 3;
 }
 
+message FragmentError {
+  FragmentErrorType error_type = 1;
+  string fragment_id = 2;
+  string detail = 3;
```

Comment:
should this be `message`?

Reply:
I think it makes sense to be a message unless I'm missing something.

Reply:
Picked `detail` just because I saw that in other protos that represent errors, no strong preference though